plugins/linux-pipewire/pipewire.c

```diff
@@ -311,6 +312,8 @@ static inline struct spa_pod *build_format(obs_pipewire_stream *obs_pw_stream,
 	struct spa_fraction min_framerate;
 	struct spa_fraction framerate;
 
+	spa_pod_dynamic_builder_init(&pod_builder, NULL, 0, 4096);
```

Comment:
This should be a lot smaller since it only needs to hold one format now instead of 50 formats with 50 modifiers it just needs to hold the 50 modifiers. I assume it grows automatically, but we can just see what the size is on amd and use that I guess.